wiki/Multi/en.md

```diff
@@ -0,0 +1,422 @@
+
+<!-- External links used -->
+[Release b335]: https://osu.ppy.sh/forum/p/26660 "osu! Public Release b335 announcement forum page."
+[osu!academy Ep6]: https://www.youtube.com/watch?v=cyYRl-a5xII "osu!academy Episode 6 youtube link."
+[Match History link]: http://osu.ppy.sh/mp/x
+[modified version link]: https://osu.ppy.sh/forum/t/135726 "ppy's reply on a bug report."
+
+<!-- Image links used -->
+[Beatmaps_peppy image]: /wiki/shared/Beatmaps_peppy.jpg "Official image on beatmap extraction."
+[Multi_lobby image]: ./img/Multi_lobby.jpg "Example of a typical day in the Multi's Lobby"
+[Multi_new_game image]: ./img/Multi_new_game.jpg "Setting up configuration for a New Game"
+[Multi_roomhost image]: ./img/Multi_roomhost.jpg "Example of Host Match Setup"
+[Multi_roomplayer image]: ./img/Multi_roomplayer.jpg "Example of Player Match Setup (With Free Mods on)"
+[Multi_mods_host image]: ./img/Multi_mods_host.jpg "Host (osu!) mods options"
+[Multi_mods_player image]: ./img/Multi_mods_player.jpg "Player (osu!) mods options (With Free Mods on)"
+[Multi_beatmapinfo image]: ./img/Multi_beatmapinfo.jpg "Beatmap information when hovered over."
+[Multi_host_song_change image]: ./img/Multi_host_song_change.jpg "When the Host is changing songs, this bar will become a temporary placeholder."
+[Multi_SS image]: ./img/Multi_SS.jpg "Example of how the Song Selection will look like for host."
+[Multi_mh_single image]: ./img/Multi_mh_single.jpg "Example of Match History."
+[Multi_mh_team image]: ./img/Multi_mh_team.jpg "Example of Team Match History."
+
+<!-- Team Mode images -->
+[HTH Interface image]: ./img/Multi_HTH.jpg "Head-to-Head Interface"
+[Team Vs Interface image]: ./img/Multi_team_vs.jpg "Team Vs Interface"
+[Tag Coop Interface image]: ./img/Multi_tag_co-op.jpg "Tag Coop Interface"
+[Tag Team Vs Interface image]: ./img/Multi_tag_team_vs.jpg "Tag Team Vs Interface"
+[HTH Grade image]: ./img/Multi_grade.jpg "Head-to-Head grade screen"
+[Team Vs Grade image]: ./img/Multi_team_grade.jpg "Team Vs grade screen"
+[Tag Coop Grade image]: ./img/Multi_co-op_grade.jpg "Team Co-Op grade screen"
+[Tag Team Vs Grade image]: ./img/Multi_tag_team_grade.jpg "Tag Team Vs grade screen"
+
+<!--Title-->
+# Multi
+
+_Multi_ was implemented on the 3rd of June 2008 (3.6.08) under [osu! Public Release b335][Release b335], and is a game mode in which up to sixteen players can compete against each other on a single map decided by the host.
+
+The [osu!academy][osu!academy wikilink] covered this page in [Episode 6 (6:52)][osu!academy Ep6] along with the [Online Users Panel][Chat_Console#Extended_Chat Console wikilink].
+
+## How to enter
+
+### Minimum requirements
+
+![One of the ways to transfer the beatmap without leaving the room][Beatmaps_peppy image]
+
+_One of the ways to transfer the beatmap without leaving the room_
+
+- Decent online connection.
+- _osu!account_ logged into _Bancho_ (should be done automatically during in-game login)
```

Comment:
The act of logging in the account (except for those that enabled the save username and password buttons).

I assume nobody save the password.